packages/webdriverio/src/commands/browser/react$$.js

```diff
@@ -15,7 +15,8 @@
     });
  * </example>
  *
- * @alias browser.react$
+ * @alias browser.react$$
+ * @param {String=} selector React elements or HTML elements
  * @param {Object=} props  React props the element should contain
  * @param {Object=} state  React state the element should be in
  * @return {Element}
```

Comment:
does it return one Element, I suppose it should be `Element[]`

Reply:
@CrispusDH  Yes, it should be `Element[]`, it returns an array of elements. I will do the changes.
@baruchvlz Please confirm.